parm/config/config.base.emc.dyn

```diff
@@ -244,7 +250,11 @@ export FHMAX_GFS=$(eval echo \${FHMAX_GFS_$cyc})
 export FHOUT_GFS=${FHOUT_GFS:-3}
 export FHMAX_HF_GFS=${FHMAX_HF_GFS:-0}
 export FHOUT_HF_GFS=${FHOUT_HF_GFS:-1}
-export STEP_GFS="24"
+if (( gfs_cyc != 0 )); then
+    export STEP_GFS=$(( 24 / gfs_cyc ))
```

Comment:
Hopefully users don't change `gfs_cyc` in their EXPDIR config.base to anything except 0, 1, 2, or 4. The `setup_expt.py` script already constrains this for generating the EXPDIR config.base but nothing stops the user from manually changing it in their generated config.base (e.g. gfs_cyc=3) and having `STEP_GFS` equal 8 (24/3). Do we want to check gfs_cyc = 0, 1, 2, or 4 on line 253 as well or instead?

Reply:
Outside the scope of this PR. All this is doing is fixing the STEP_GFS calculation. This was just a mistake when brining CROW stuff over. Later, I plan to eliminate gfs_cyc entirely and just set STEP_GFS instead.